omp/solver/idr_kernels.cpp

```diff
@@ -33,10 +33,10 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include "core/solver/idr_kernels.hpp"
 
 
-#include <time.h>
 #include <algorithm>
 #include <iostream>
 #include <random>
+#include <time.h>
```

Comment:
Maybe this should be in a separate section of its own as it is a C header ?

Reply:
I guess the correct include here would be `ctime`, which should definitely be part of the same group, or even use std::random_device, but I wanted to keep the changes as minimal as possible. Maybe something for a later PR?